graylog2-web-interface/src/components/users/UserCreate/UserCreate.tsx

```diff
@@ -187,7 +196,12 @@ const UserCreate = () => {
               </div>
               <div>
                 <Headline>Settings</Headline>
-                <TimeoutFormGroup />
+                {isGlobalTimeoutEnabled ? (
+                  <GlobalTimeoutMessage label="Sessions Timeout"
```

Comment:
The timeout disabled box is not centered vertically compared to the other fields. I think we should fix it. see below 
![image](https://user-images.githubusercontent.com/1381455/213634244-2355b40b-b1ef-4353-ba6d-b7af611eb8d4.png)

Reply:
yes there was an extra padding top there, should be fixed now.